From abb9063da6380f316c90801456af227f24e8521b Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Thu, 19 Sep 2019 13:52:23 +0000
Subject: [PATCH] OS-7993 build_live bi_gen_version_files missing a pfexec
 Reviewed by: Alex Wilson <alex@cooperi.net> Approved by: Tim Foster
 <tim.foster@joyent.com>

---
 tools/build_live | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/build_live b/tools/build_live
index 103d8a98..ed5528d4 100755
--- a/tools/build_live
+++ b/tools/build_live
@@ -428,7 +428,7 @@ function bi_gen_version_files
 	    fail "failed to copy gitstatus"
 
 	if ! cmp "$gitstatus" "$bvfile" >/dev/null 2>&1; then
-		cp "$gitstatus" "$bvfile" ||
+		pfexec cp "$gitstatus" "$bvfile" ||
 		    fail "could not copy build version"
 	fi
 
-- 
2.21.0

